doc/format.rst

```diff
@@ -250,13 +250,21 @@ The sections of a function's docstring are:
    Support for the **Yields** section was added in `numpydoc
    <https://github.com/numpy/numpydoc>`_ version 0.6.
 
-7. **Other Parameters**
+7. **Sent**
```

Comment:
I am a bit torn about the order of Yields and Sent.  Putting Sent first feels more natural to the way you use `send` (in that the next value to come out probably depends on the value you sent in or you would not be bothering to send anything in!).

On the other hand it yields a non-trivial value before you can send in a non trivial value so it should stay in this order.

On the other other hand, you do send in `None` first to prime the generator (but that is not what is being documented in this block).

Reply:
I had thought the idiom was to use `next` initially, then `.send` later.

I think the discomfort is partially with the sense of the word "Sent", which connotes a departing aspect. "Receives", in contrast, seems to fit better after "Yields".

Reply:
Interesting, I had not seen that idiom. I like sending `None` as it special-cases the first round less, but that is neither here nor there for this discussion.

"The generator is sent ..." vs "The generator receives ..."

The first is odd because we drop the 'is' in the section title and the second is odd because it feel more passive.

Reply:
"receives" sounds more correct than "sent" in my head.

Reply:
I prefer "receives" as well. Another possibility I can think of is "accepts," but I don't think I like that as much as "receives".